.github/workflows/ci.yml

```diff
@@ -23,7 +23,6 @@ jobs:
         platform:
         - ubuntu-latest
         go:
-          - 1.16.x
```

Comment:
I may need to think about this CI change a bit more. It'd be ideal if we could just skip 1.16 only for `gopages`, that way all the other tools are still tested against 1.16. That darn `x/sys` (indirect) dependency is always causing trouble across Go versions.

Reply:
Do you need something from me on this ?

Reply:
I think this is okay after all. Gopages will have a higher Go compatibility requirement than the others, it's just a bummer we can't run the tests (easily) for all modules on 1.16.